cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/env/ReturnValues.java

```diff
@@ -32,4 +32,13 @@ public ReturnValues(Map<String, Serializable> outputs, String result) {
     public String getResult() {
         return result;
     }
+
+    @Override
+    public String toString() {
```

Comment:
In the logs appeared an object reference before, I thought it would be better to see the actual values. WDYT?